frontend/src/pages/surveys/libs/constants.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  PREFERENCES_OTHER_CATEGORY,
+  type SurveyInputDto,
+} from '#packages/survey/survey.js';
+
+const TEXTAREA_MAX_LENGTH = 1000;
+const TEXTAREA_ROWS_COUNT = 3;
+
+const INVALID_ARRAY_INDEX = -1;
```

Comment:
It's a common constant

Reply:
@v0ldemar01 Should I create constants folder in `src/libs`?

Reply:
Done